internal/elasticsearch/security/user_data_source.go

```diff
@@ -63,10 +63,9 @@ func DataSourceUser() *schema.Resource {
 }
 
 func dataSourceSecurityUserRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	var diags diag.Diagnostics
-	client, err := clients.NewApiClient(d, meta)
-	if err != nil {
-		return diag.FromErr(err)
+	client, diags := clients.NewApiClient(d, meta)
```

Comment:
These look like they were fun :)

Reply:
Haha well there was something satisfying about checking them off and have the files turn green